go/oasis-test-runner/oasis/network.go

```diff
@@ -45,6 +45,11 @@ import (
 	staking "github.com/oasisprotocol/oasis-core/go/staking/api"
 )
 
+const (
+	cfgMetricsAddr     = "metrics.address"
```

Comment:
Perhaps, but I think this is out of scope for this PR.  The metrics code is a mess and requires a larger overhaul.

Reply:
Yeah this does seem a bit hackish and the proposal from @peternose seems like it would be a quick fix, e.g.:
* Adding a `MetricsCfg` structure to `NetworkCfg` (it can just have these two fields, e.g. address and interval).
* Just populate it from viper in `doScenario` (in `cmd/root.go`) where you have these already available.
* And then below in `startOasisNode` it can be just configured from `net` instead of using `viper`.

Reply:
Ah, you're right, this wasn't as bad as I've imagined -- it's fixed now :)